.gitignore

```diff
@@ -3,3 +3,4 @@ repository
 repository-x86_64
 */*/download
 */*/work-*
+/haiku-apps/bemobile/additional-files
```

Comment:
Hmm, this PR should not modify the `.gitignore` file.
Are you using a "high-level" git front-end?

Reply:
Yes sir.